src/repositories/KeyReleasePolicyStore.ts

```diff
@@ -0,0 +1,134 @@
+ // Copyright (c) Microsoft Corporation.
+ // Licensed under the MIT license.
+ 
+ import * as ccfapp from "@microsoft/ccf-app";
+ 
+ export class KeyReleasePolicyStore<K extends string, T> {
+   private _store;
+   private _claims = {
```

Comment:
It looks like you can turn this into an interface or type, might be easier?

```typescript
interface ClaimsInterface {
    "x-ms-attestation-type": string,
    "x-ms-compliance-status": string,
...
}
```
This interface can be used directly with the CCF map: `const kv = ccfapp.TypedKvMap<string, ClaimsInterface>`.

You can then replace the `records<string, any>` type with the `ClaimsInterface`.  The only validation needs to be done at the start of each endpoint, where you probably want a type guard to go from a `ccf.Request` body to a `ClaimsInterface`.